.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,24 +1,16 @@
 ---
 name: Bug report
-about: Create a report to help us improve
```

Comment:
This file is meant to be used automatically by GitHub when submitting a bug report issue, no need to commit any changes to it when fixing bugs.